docs/en/observability/uptime-set-up-app.asciidoc

```diff
@@ -0,0 +1,168 @@
+// instead of what is _not_? 
+With Elastic's global managed testing infrastructure, you can create and run monitors in multiple
+locations using the {uptime-app} the UI without having to set up {fleet},
+pull a Docker image, and install and run an agent on your machine.
+
+To add a monitor using Elastic's global managed testing infrastructure:
+
+. Go to the **Monitor Management** page.
+. Click **Add monitor**.
+. In _Monitor locations_, select one or more locations.
+. Choose a _Monitor type_ and configure the monitor as needed.
+. Click **Save monitor**.
+
+If you are setting up a **Browser** synthetic monitor, see <<synthetics-create-test>>
+for more details on writing synthetic tests.
+
+[discrete]
+[[private-locations]]
+== Run a monitor on private locations
+
+Private locations allow you to run monitors from your own premises in addition to or
+instead of on Elastic's global managed testing infrastructure.
+
+// The TL;DR of what follows
+Before you create a private location, you need to set up {fleet} and {agent},
+pull a Docker image, and connect to the {stack}. Then, you can create a private location
+in the {uptime-app} UI so when you add a new monitor, the private location will be
+available in the list of _Monitor locations_.
+
+[discrete]
+[[private-locations-fleet-agent]]
+=== Set up {fleet-server} and {agent}
+
+// Is this sufficient or should we copy steps for setting up
+// Fleet and Agent into this page directly?
+If want to run monitors on private locations, there are two prerequisites to complete before you can configure monitors:
+
+* *Set up {fleet-server}*: If you are using {ecloud}, {fleet-server} will already be provided and you can skip this step.
+To learn more, see <<set-up-fleet,{fleet}>>.
+* **Create an agent policy**:  For more information on agent policies and creating them,
+see {fleet-guide}/agent-policy.html#create-a-policy[{agent} policy].
+
+[discrete]
+[[private-location-docker]]
+=== Pull the Docker image
+
+Elastic provides Docker images that you can use to run monitors, which reduces the chance of problems.
+
+For monitors running on private locations, you _must_ use the `elastic-agent-complete`
+Docker image to create a self-hosted {agent} node. The standard {ecloud} or self-hosted
+{agent} will not work.
+
+// ifeval::["{release-state}"=="unreleased"]
+
+// Version {version} has not yet been released.
+
+// endif::[]
+
+// ifeval::["{release-state}"!="unreleased"]
+
+To pull the Docker image run:
+
+[source,sh,subs="attributes"]
+----
+docker pull docker.elastic.co/beats/elastic-agent-complete:{version}
+----
+
+// endif::[]
+
+[discrete]
+[[private-locations-connect]]
+=== Connect to the {stack}
+
+After configuring the monitor, run it in Docker and connect the monitor to the {stack}.
+To add an {agent} to {fleet}, you'll need an enrollment token and the URL of your {fleet-server}.
+You can use the default enrollment token for your policy or create new enrollment tokens as needed.
```

Comment:
I think saying that one runs the monitor in docker and connects it to the stack is a bit confusing/inaccurate.

What you actually need to connect to the stack is Fleet, and then enroll Agents, so I updated it to reflect that. Lmk what you think:

```suggestion
Before you can create a private location, you must connect {fleet} to the {stack}.

Then, to enroll an {agent} to {fleet}, you'll need an enrollment token and the URL of the {fleet-server}.

You can use the default enrollment token for your policy or create new policies and enrollment tokens as needed.
```

Reply:
Ahh thanks for clarifying.